File: 2.0/java/numbers/updateNumberLease.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.numbers.model.NumberLease;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient client = new CallfireClient("api_login", "api_password");
        NumberLease lease = new NumberLease();
        lease.setNumber("19206596476");
        lease.setAutoRenew(false);
        lease.setCallFeatureStatus(NumberLease.FeatureStatus.DISABLED);
        client.numberLeasesApi().update(lease);
    }
}

File: 2.0/csharp/numbers/updateNumberLease.cs
using CallfireApiClient;
using CallfireApiClient.Api.Numbers.Model;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        var lease = new NumberLease
        {
            PhoneNumber = "19206596476",
            AutoRenew = false,
            CallFeatureStatus = NumberLease.FeatureStatus.DISABLED
        };
        client.NumberLeasesApi.Update(lease);
    }
}
